config/routes.rb

```diff
@@ -233,6 +233,7 @@
         get :manage
       end
     end
+    resources :kudos, only: [:index]
```

Comment:
This is a more efficient syntax for something that only has one entry, but it's not consistent with the rest of the routes file. Should we update the others to the new way, or keep this the same as the others for now?